pylabnet/utils/helper_methods.py

```diff
@@ -825,6 +832,12 @@ def launch_script(script, config, log_ip, log_port, debug_flag, server_debug_fla
     cmd += f'--server_debug {server_debug_flag}'
     cmd += client_cmd
 
+    if len(cmd) > 8191:
+        if logger is not None:
+            logger.error('Cmd too long! Server will not instantiate!')
```

Comment:
Maybe we should just return here or raise an exception, otherwise the `Popen` command will just crash.

Reply:
We can, but the end result is the same (since Popen fails silently)

Reply:
Oh, I meant just return or raise in addition to outputting the `log.error`. That way we avoid launching `Popen` knowing that it will fail.

Reply:
Added